static/js/app.js

```diff
@@ -178,30 +69,30 @@ async function addLifeEvents() {
       ${event.more ? `<p>More: ${event.more}</p>` : ''}
     </div>
   `).join('');
-  console.log("life events",lifeEventsHtml)
+  console.log(`${eventType} events`, eventsHtml)
   // Append the generated HTML to the container
-  lifeEventsContainer.innerHTML = lifeEventsHtml;
+  eventsContainer.innerHTML = eventsHtml;
 }
 
 ////////////////////////Execute methods\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
-//DOM Listener makes sure all the content is loaded befere execute the methods
+//DOM Listener makes sure all the content is loaded before execute the methods
 document.addEventListener('DOMContentLoaded', async () => {
-  // loads About me page
-  await loadAboutMePage();
+  // // loads About me page
+  await loadEventHtml("aboutMe");
   // Call the function to load 'experience.html' and execute subsequent logic
-  await loadExperiencePage();   
+  await loadEventHtml("experience");
   // call function to add experience events
-  addExperienceEvents();
+  await addEvents("experience","Work");
```

Comment:
Whenever you call a function with multiple parameters, sometimes you add a space before the comma, and sometimes you don't. I believe you should be more consistent. To help you so, look at the `eslint` section about commas: https://eslint.org/docs/latest/rules/comma-spacing#options

Remember that you can configure an .eslint file for your project so that all major IDEs would be able to highlight your formatting: https://eslint.org/docs/latest/use/getting-started. 

(In a personal project like this one it might make little sense, but in larger projects, you should avoid everyone coding in their own style)